include/librealsense2/hpp/rs_frame.hpp

```diff
@@ -50,20 +83,32 @@ namespace rs2
                     fps() == rhs.fps();
         }
 
+		/**
+		* Template function, checking if the instance belongs to specific class type
+		* \return bool - true or false.
+		*/
         template<class T>
         bool is() const
         {
             T extension(*this);
             return extension;
         }
 
+		/**
+		* Template function, changing the instance as another class type
```

Comment:
Casting is the more appropriate term. the original type does not change during this operation